redaxo/src/core/assets/standard.js

```diff
@@ -703,12 +703,28 @@ jQuery(document).ready(function ($) {
                 window.location = 'index.php?page=login';
                 break;
             default:
-                // load URL and show error/whoops
+                // load URL (could be error/whoops also)
                 window.location = event.request.responseURL;
                 break;
         }
     });
 
+    // handle pjax response, https://github.com/MoOx/pjax#handleresponseresponsetext-request-href-options
+    pjax._handleResponse = pjax.handleResponse;
+    pjax.handleResponse = function(responseText, request, href, options) {
+        if (request.getResponseHeader('content-disposition').indexOf('attachment') !== -1) {
```

Comment:
Bilder und PDFs liefere ich normalerweise eher als `inline` statt `attachment` aus. Daher glaube ich, passt dieses Kriterium auch noch nicht richtig.
Bzw. es reicht noch nicht. Ich würde es als ein Kriterium lassen, also wenn attachment, dann auf jeden Fall neu laden.

Reply:
Ich habe es jetzt so angepasst, dass PJAX nicht dynamisch lädt bei folgender Bedingung:

- Content-Disposition ist `attachment` oder
- Content-Disposition ist `inline` und Content-Type ist nicht `text/html`

Damit sollte es klappen, so scheint mir. Was hälst du davon?

Reply:
Ich würde den inline-check eigentlich eher weglassen.
[Hier](https://github.com/redaxo/redaxo/blob/master/redaxo/src/addons/be_style/plugins/customizer/boot.php#L17-L51) liefern wir als Beispiel eine CSS aus, ohne Content-Disposition, aber mit Content-Type. Konkret in dem Fall wird die Datei eher nicht irgendwo verlinkt werden, also konkret die Datei wird somit natürlich nicht vorkommen.
Aber trotzdem denke ich eigentlich, wenn doch Content-Type mit etwas anderem belegt ist, dann reicht das doch, dass es kein html ist.

Reply:
Ja, ist mir auch recht. Hab’s nochmal angepasst.